### PR TITLE
feat: listing hidden channel first version

### PR DESCRIPTION
**Pull request by america-nishi**
This are the list for the hidden channels. I will keep working but with the new changes in main.

### CODE REVIEW

File: src/features/channels/components/channel-list/search-channel/SearchChannel.tsx [hunk 1]
```diff
@@ -69,14 +92,35 @@ const ChannelList = () => {
...
             </ListItem>
+
+            <Modal open={open} onClose={closeModal} title="Hidden Channels">
+              <Card sx={{ minWidth: 400 }}>
+                {hiddenChannel.map(
+                  (channel) =>
+                    !channel.visible && (
+                      <Box>
+                        <Divider />
+                        <Box className={styles.cardContainer}>
```
Comment (josetheis-stackit) [inline src/features/channels/components/channel-list/search-channel/SearchChannel.tsx:103]: Remember that when iterating through and array and rendering the items of that array you need to specify a 'key' prop in the parent component of what you're returning in the 'map' function. If you look at the console right now you'll get a nasty error saying you're missing the 'key' prop.

**Review by josetheis-stackit — CHANGES_REQUESTED** [review]
Left a comment requesting a change, please take a look.

File: src/features/channels/components/channel-list/search-channel/SearchChannel.tsx (hunk 1 above)
Comment (america-nishi) [reply src/features/channels/components/channel-list/search-channel/SearchChannel.tsx:103]: ok, thank you. I will make the changes

**Review by josetheis-stackit — APPROVED** [review]
LGTM

**Pull request merged**